yeat/tests/test_bandage.py

```diff
@@ -15,37 +15,31 @@
 import subprocess
 from unittest.mock import patch
 from yeat.tests import data_file
-from yeat.workflows import bandage
+from yeat.workflows import snakefiles
```

Comment:
Could you explain your thinking on introducing a "snakefiles" subsubpackage here? The "workflows" subpackage is almost empty except for "snakefiles", which suggests the new layer of nesting might be unnecessary.